api/integrations/google/gcal-bot

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import datetime
+import httplib2
+import itertools
+import logging
+import optparse
+import os
+from six.moves import urllib
+import sys
+import time
+import traceback
+from typing import List, Set, Tuple, Iterable, Optional
+
+from oauth2client import client, tools
+from oauth2client.file import Storage
+try:
+    from googleapiclient import discovery
+except ImportError:
+    logging.exception('Install google-api-python-client')
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../'))
+import zulip
+
+SCOPES = 'https://www.googleapis.com/auth/calendar.readonly'
+CLIENT_SECRET_FILE = 'client_secret.json'
+APPLICATION_NAME = 'Zulip'
+HOME_DIR = os.path.expanduser('~')
+
+# Our cached view of the calendar, updated periodically.
+events = [] # type: List[Tuple[int, datetime.datetime, str]]
```

Comment:
I prefer (together with pep8 standard) to have 2 spaces between end fo the code and comment `#` - could you fix all of the comments. I mean only comments that are in the same line with code